public/session-storage.js

```diff
@@ -0,0 +1,83 @@
+(function collectAnswers () {
```

Comment:
At the top of this file:
```js
/* global: location, sessionStorage */
```
http://eslint.org/docs/user-guide/configuring#specifying-globals